cleanrepo/CleanRepo/Options.cs

```diff
@@ -2,26 +2,32 @@
 
 namespace CleanRepo;
 
-// Define a class to receive parsed values
+// Define a class to receive parsed values.
 class Options
 {
-    [Option("start-directory", HelpText = "Top-level directory in which to perform clean up (for example, find orphaned markdown files).")]
-    public string InputDirectory { get; set; }
+    [Option("docfx-directory", HelpText = "Directory that contains the docfx.json file of interest.")]
```

Comment:
All of these option name updates will have to be changed in the _action.yml_ as well. And if you want to change the mapping in what the GitHub Action accepts as input, you'd have to update the consuming workflows too.